### ISSUE TITLE
Newbie Question - installation, features/benefits of the app

### ISSUE THREAD

**Issue by benfrankie** [gophish/gophish#2342]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?:  gophish-v0.11.0-windows-64bit.zip

Brief description of the issue: unable to install
![image](https://user-images.githubusercontent.com/35661604/140005484-dc7bbf88-d883-4f0b-bdd2-702b83d7a5ab.png)

What are you expecting to see happen? : install 

What are you seeing happen? error

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:

```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - downloaded, tried installing following this instruction https://www.youtube.com/watch?v=lI6rCBYGNF8&t=197s 
 Windows 7 
![image](https://user-images.githubusercontent.com/35661604/140005920-a8546dfa-c048-4e70-9dc6-7e359d429498.png)

 * Step 2 - 
 
hello, can you  please help share a solution? 
Fyi I'm not technical so a non-tech friendly instruction would be greatly appreciated. 

Also wondering if GoPhish is a right solution for inboxing cold emails in bulk? 
My goal is to inboxing cold emails in bulk - as many as possible like 300k+ emails/day. 
If this is possible, what would be the requirements? i.e. system specs, smtp (is one gmail smtp gonna be good enough or smtp services like sendgrid is required? ) 

based on the technical terms on your website, I'm not clearly what the benefits, use cases, limitations of this app would be exactly... 

thanks

**Comment by glennzw** [comment]
I don't think gophish is the right solution, as it's specifically a security awareness tool. I think you'd have better luck with Sendgrid or Mailchimp (as that's their business model). You'll likely get blocked by Google quite soon if you try 300k with gophish.

**Comment by benfrankie** [comment]
> I don't think gophish is the right solution, as it's specifically a security awareness tool. I think you'd have better luck with Sendgrid or Mailchimp (as that's their business model). You'll likely get blocked by Google quite soon if you try 300k with gophish.

would it be possible to use sendgrid and gophish to inbox 300k+ emails/day? the problem with SG is that the inbox rate is very low... most emails end up in spam

**Comment by edermi** [comment]
You'll have the same problem with Gophish

**Comment by glennzw** [comment]
Gophish needs to send emails via something - in this case you're asking to send via SendGrid, but you can just send directly with sendgrid (and you're already being sent to spam). I don't think gophish is the right tool for you.
